app/views/posts/_form.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= form_with model: @post, url:user_posts_path, method: :post, local: true, scope: :post do |f| %>
+  <div>
+    <%= f.label :title %><br>
+    <%= f.text_field :title %>
+  </div>
+  <div>
+    <%= f.label :text %><br>
+    <%= f.text_field :text %>
+  </div>
+   <%= f.submit "Add Post" %>
+<% end %>
```

Comment:
- According to the project requirements, There should be a form in the views/posts/new.html.erb file which accepts input for creating posts. I suggest creating the view page that renders your form. please fix this 🙏 
- According to the project requirements, A comment is created on behalf of the currently logged in user (current_user) of the app. The comments controller should have new and create actions similar to the one shown below. Your project does not have a comments controller setup. Please fix this 🙏 
- Also, note that just as with the posts, There should be a form in the views/comments/new.html.erb file which accepts input for creating comments. Please fix this 🙏